### ISSUE TITLE
[BUG] Every error results in "I'm sorry, an internal error has occurred."

### ISSUE THREAD

**Issue by uvnikgupta** [NVIDIA/NeMo-Guardrails#512]
Ln : <code>368</code> to <code>372</code> or <code>colang/v1_0/runtime/runtime.py</code>
```python
if status == "failed":
      # TODO: make this message configurable.
      result = self._internal_error_action_result(
          "I'm sorry, an internal error has occurred."
      )
```

This is very frustrating as it hides all exceptions and make it very difficult to debug and issue. Please remove this and propagate the exception. Let the client decide what to do with the exception.

**Comment by drazvan** [comment]
I agree. We should make this configurable. This solution is the recommended one by the security team. This significantly reduces the chance of exposing sensitive information on errors.
